app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace App\Http\Controllers;
+
+
+use Illuminate\Http\Request;
+use App\Post;
+use App\Models;
+use Illuminate\Support\Facades\Auth;
+
+class PostController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(){
+    
+        $posts =\App\Models\Post::all();
+        
+        return view('top',["post" => $posts]);
+    
+        return response()->json([
+            'posts' => $posts
+        ]);
+    }
+    
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create(Request $request) 
+    {
+       
+        $input = $request->only('message','user_id');
+		
+        $entry = new Models\Post();
+        
+       
+    }
+    public function detail(Post $post)
+    {
+       return view('posts/detail', [
+          
+           'content' => $post->message,
+           'user_id' => $post->user_id,
+       ]);     
+         
+    }
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        
+       $entry = new Models\Post();
```

Comment:
PostModelのインスタンスを保持する変数なので、変数名も$postにしましょう
```suggestion
       $post = new Post();
```